tree/20_cfe_basics/ncf_lib.cf

```diff
@@ -862,30 +840,31 @@ body classes classes_generic_return_boolean_list_two(x, y, boolean_prefix, true_
     repaired_returncodes => { @{false_return_codes} };
 
     promise_repaired => { "promise_kept_$(x)", "$(x)_kept", "$(x)_ok", "$(x)_not_repaired", "$(x)_reached",
-                          "promise_kept_$(y)", "$(y)_kept", "$(y)_ok", "$(y)_not_repaired", "$(y)_reached", "${boolean_prefix}_false" };
-    repair_failed => { "repair_failed_$(x)", "$(x)_failed", "$(x)_not_ok", "$(x)_not_kept", "$(x)_not_repaired", "$(x)_reached", "${x}_error",
```

Comment:
why removing these?

Reply:
Wrong copy/paste, I forgot the inherit_from part